assets/js/steps/components/input.spec.js

```diff
@@ -7,96 +7,130 @@ import React from 'react';
 import { shallow, mount } from 'enzyme';
 import { expect, it, describe } from '@jest/globals';
 import Input from './input';
+import { IntlProvider } from 'react-intl';
+
+const translations = {
```

Comment:
Are these translations the same across tests? Then they should be moved to file that can be imported instead.

Reply:
Might have been fixed.